docker-compose.yml

```diff
@@ -6,10 +6,11 @@ services:
     - 27017:27017
 
   web:
+    platform: linux/amd64
```

Comment:
Is this change really necessary?
What is the error if you don’t specify the platform?